src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetTemplate/index.spec.tsx

```diff
@@ -1,9 +1,10 @@
 import { act, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import React from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
 
 import { wrapInIntlProvider } from 'utils/tests/intl';
```

Comment:
Can be removed I think, you don't use `useIntl` anymore.

Reply:
yes, I deleted it